sdks/java/io/rrio/src/main/java/org/apache/beam/io/requestresponse/Cache.java

```diff
@@ -27,13 +27,28 @@
 import org.apache.beam.sdk.transforms.PTransform;
 import org.apache.beam.sdk.values.KV;
 import org.apache.beam.sdk.values.PCollection;
+import org.apache.beam.vendor.guava.v32_1_2_jre.com.google.common.annotations.VisibleForTesting;
 import org.apache.beam.vendor.guava.v32_1_2_jre.com.google.common.io.ByteSource;
 import org.checkerframework.checker.nullness.qual.Nullable;
 import org.joda.time.Duration;
 
 /** Transforms for reading and writing request/response associations to a cache. */
 final class Cache {
 
+  /**
+   * {@link VisibleForTesting} to test {@link RequestResponseIO} composite transform construction
+   * without exposing {@link UsingRedis.Read}.
+   */
+  @VisibleForTesting
+  static final String CACHE_READ_USING_REDIS = Cache.UsingRedis.Read.class.getName();
```

Comment:
Were you able to sync with @kennknowles on redis usage?

Reply:
Scheduled meeting with him.

Reply:
Resolving this as we discussed. Please re-open if this remains an issue. Thank you for looking at this PR.